[[source]]
name = "pypi"
url = "https://pypi.org/simple"
verify_ssl = true

[dev-packages]
flake8 = "==3.8.3"
flake8-bugbear = "*"
flake8-builtins = "*"
pytest = "*"
pytest-asyncio = "*"
pytest-cov = "*"
requests = "*"
snapshottest = "*"
sqlalchemy-utils = "*"
wasmer = "*"

[packages]
aiosmtplib = "==1.1.*"
alembic = "==1.7.*"  # SQL migrations
cryptography = "==3.0.*"  # controller password encryption
cffi = "==1.14.*"
gino = "==1.0.*"
gino-tornado = "==0.1.*"
graphene = "==2.1.*"
graphene_tornado = "==2.6.*"
ldap3 = "==2.9.*"  # ldap integration
psycopg2-binary = "==2.8.*"  # alembic migrations
pyjwt = "==1.7.*"  # JWTTokens
pymemcache = "==3.5.*"
pyotp = "==2.6.*"
python-dateutil = "==2.8.*"
python-ldap = "==3.4.*"  # ldap integration
python2-secrets = "==1.0.*"  # Django-secrets
SQLAlchemy = "==1.3.*"
tornado = "==6.1.*"
uvloop = "==0.16.*"
veil-api-client = "==2.2.*"
veil-aio-au = "==0.1.*"
yaaredis = "==2.0.*" # aioredis и aredis имеют баги. aioredis не дает создать больше одного подписчика на канал. aredis не пробрасывает asyncio.CancelledError

[requires]
python_version = "3.7"
